ml/misc/interview.py

```diff
@@ -26,3 +31,79 @@ def get_2nd_largest(num_list):
     if counter < 2:
         return None
     return m2nd
+
+
+def get_fibonacci(n):
+    """
+    return Nth fibonacci number. Advanced: can you do BigO(1)?  O = O(n)
+    Since fibonacci function is (A**n-B**n)/(5**(0.5)), where A = (1+5**1/2)/2, and B = (1-5**(1/2))/2,
+    therefore it has to be O(n)
+    :param n: Nth fibonacci number, ints
```

Comment:
In PyCharm, go to Preferences -> Tools -> Python Integrated Tools -> Docstrings, choose "Epytext" (instead of reStructuredText) as format, and change all `:` to `@` in your docstring